stream/clients/python/setup.py

```diff
@@ -31,7 +31,7 @@
     'six>=1.10.0',
     'pytz',
     'futures>=3.2.0;python_version<"3.2"',
-    'grpcio<1.28,>=1.8.2',
+    'grpcio>=1.37',
```

Comment:
My understanding is that we have to put a cap here  otherwise  auto upgrades may break the client.
IIRC it happened in the past.

I I am mot sure that we have CI that is covering this change.

Reply:
Thank you! I threw the change here to see if it breaks anything. To prepare a solid patch I need more time to understand the mechanism and see how to verify the change.